<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Homepage</class>
 <widget class="QMainWindow" name="Homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>运动健康中心</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="profileFrame">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string>background-color: #f8f9fa; border-radius: 10px; padding: 15px; border: 1px solid #e0e0e0;</string>
      </property>
      <layout class="QVBoxLayout" name="profileLayout">
       <item>
        <widget class="QLabel" name="avatarLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 40px; border: 2px solid #3498db; background-color: #e0e0e0;</string>
         </property>
         <property name="text">
          <string>头像</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>张三</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>身高: 175 cm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weightLabel">
         <property name="text">
          <string>体重: 65 kg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bmiLabel">
         <property name="text">
          <string>BMI: 21.2 (正常)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutButton">
         <property name="styleSheet">
          <string>background-color: #3498db; color: white; border-radius: 5px; padding: 8px;</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="historyFrame">
      <property name="styleSheet">
       <string>background-color: white; border-radius: 10px; padding: 15px; border: 1px solid #e0e0e0;</string>
      </property>
      <layout class="QVBoxLayout" name="historyLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>历史运动数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="chartView" native="true">
         <property name="styleSheet">
          <string>background-color: #fafafa; border: 1px dashed #cccccc; border-radius: 5px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
